### ISSUE TITLE
MRBS 1.11.4/3 - 500 - Internal server error for Approve booking

### ISSUE THREAD

**Issue by HONGNGUYEN2023** [meeting-room-booking-system/mrbs-code#3545]
I am using room booking version MRBS 1.4.11. It is working fine with the Approval booking. We need to migrate to new server so I have downloaded MRBS 1.11.4/3 and config with the new version. It is working fine except approval function. Admin can approve and email can send to the booker however the admin got an error message: "500 - Internal server error. There is a problem with the resource you are looking for, and it cannot be displayed."
I tried to check however I don't know how to fix the error message.

**Browser details (please complete the following information):**
 
 - Browser Version 118.0.5993.71 
 - Version: Chrome

**Server details (please complete the following information):**
 - MRBS version MRBS 1.11.4/MRBS 1.11.3
 - Web server: [e.g. Apache 2.4.48]
 - OS: Window server 2019
 - PHP version: php-8.2.11
 - Database used: [e.g. MySQL 5.7.43.0]

**Additional context**

**Comment by campbell-m** [comment]
You need to look in your PHP error log to find the detailed error message.

**Comment by HONGNGUYEN2023** [comment]
Hi Camp;
Could you please advise how to get PHP error log. I have tried to do it is not working.
Thanks, Hong Nguyen

**Comment by campbell-m** [comment]
Your PHP error log is normally in a file called error.log in your MRBS directory on Unix servers or C:\Windows\Temp\php-errors.log on Windows.  You can find out where it is by running `phpinfo()`.

If you still can't find it then you can set `$debug = true;` in your MRBS config file and errors will be displayed in your browser.  But remember to turn this off for a production site.

**Comment by HONGNGUYEN2023** [comment]
Dear Campell;

I cannot check the error log as you advise. the my concern is any one have the same issue with us. how to debug the issue?

Thanks, Hong Nguyen

**Comment by campbell-m** [comment]
Have you tried setting `$debug = true;`?

**Comment by HONGNGUYEN2023** [comment]
Yes, I have added $debug = true; to the config file and the error message as below. 

500 - Internal server error.
There is a problem with the resource you are looking for, and it cannot be displayed.

**Comment by campbell-m** [comment]
I have just fixed a problem in [95275e6](https://github.com/meeting-room-booking-system/mrbs-code/commit/95275e691c08066dd269037aedd25351828ea3cb).  Can you try running the latest development code which you can get by following the green Code button on [this page](https://github.com/meeting-room-booking-system/mrbs-code).  Treat it like a new release.

**Comment by HONGNGUYEN2023** [comment]
Dear Campel;

I have downloaded the latest you share on meeting-room-booking-system/mrbs-code: MRBS application code (github.com)<https://github.com/meeting-room-booking-system/mrbs-code> I still get the error message after click Approve and email send to admin and booking have approved: 500 - Internal server error. There is a problem with the resource you are looking for, and it cannot be displayed => it should return to the view page as previous version


***@***.***

However, admin have received an email for approve and the booking has approved already for 1 booking or series booking.

***@***.***
***@***.***

HONG NGUYEN
BUSINESS SYSTEMS MANAGER
T  +84 (0) 28 3744 2335
M +84 (0) 906748178
BIS HCMC
246 Nguyen Van Huong Street,
Thao Dien Ward, Thu Duc City,
Ho Chi Minh City, Vietnam
[Text  Description automatically generated with medium ***@***.***

bisvietnam.com<https://www.nordangliaeducation.com/bis-hcmc?utm_source=email&utm_medium=email&utm_campaign=email_email_general_bis_hcmc&utm_term=aug_2023>

[A red circle with a white letter f in it  Description automatically generated]<https://www.facebook.com/BISHCMC/>[A red circle with a white logo  Description automatically generated]<https://www.instagram.com/bishcmc/>[A red circle with white letters on it  Description automatically generated]<https://www.linkedin.com/school/british-international-school-ho-chi-minh-city/>[A red circle with white x in it  Description automatically generated]<https://twitter.com/bishcmc>[A red circle with a white play button  Description automatically generated]<https://www.youtube.com/user/bishochiminhcity>
[A screen shot of a white sign  Description automatically generated]<https://www.nordangliaeducation.com/bis-hcmc?utm_source=email&utm_medium=email&utm_campaign=email_email_general_bis_hcmc&utm_term=aug_2023>

**Comment by campbell-m** [comment]
What have you got `$auth['type']` set to in your config file?  And have you made any changes to the MRBS code?
